docs/tutorial.md

```diff
@@ -492,16 +516,16 @@ If you do not see your metric data in New Relic Insights, please check [configur
 
 
 ### Fetching Inventory data
-The `populateInventory` function is used to fetch inventory data:
+This snippet shows where you can add inventory data:
 ```go
-func populateInventory(inventory sdk.Inventory) error {
-	// Insert here the logic of your integration to get the inventory data
-	// Ex: inventory.SetItem("softwareVersion", "value", "1.0.1")
-	// --
-	return nil
+// Add Inventory item
+if args.All() || args.Inventory {
+    // Insert here the logic of your integration to get the inventory data
+	// err = entity.SetInventoryItem("instance", "version", "3.0.1")
+    //panicOnErr(err)
 }
 ```
-Notice that in the code above we use the `Inventory` type from the SDK package for inventory data. The `SetItem` method stores a value into the inventory data structure. The first argument is the name of the inventory item, and the other two are a field name and the inventory data value.
+Notice that in the code above we use the `SetInventoryItem` method stores a value into the inventory data structure. The first argument is the name of the inventory item, and the other two are a field name and the inventory data value.
```

Comment:
Replacing the "*stores a value*" with "*to store a value*" makes the sentence more clear.  

> Notice that in the code above we use the `SetInventoryItem` method to store a value into the inventory data structure